atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -650,13 +674,167 @@ def set_build_image(self):
             raise RuntimeError("Build object is malformed, failed to fetch buildroot image")
 
         if build_kind == 'DockerImage':
-            self.config_kwargs['build_image'] = build_name
+            return build_name
         else:
             raise RuntimeError("Build kind isn't 'DockerImage' but %s" % build_kind)
 
+    def process_image_from(self, image_from):
+        build_image = None
+        imagestream = None
+
+        if image_from['kind'] == 'DockerImage':
+            build_image = image_from['name']
+        elif image_from['kind'] == 'ImageStreamTag':
+            imagestream = image_from['name']
+        else:
+            raise UnknownKindException
+
+        return build_image, imagestream
+
+    def set_build_image(self, platforms):
+        """
+        Overrides build_image for worker, to be same as in orchestrator build
+        """
+        current_platform = platform.processor()
+        orchestrator_platform = current_platform if current_platform else 'x86_64'
+        status = get_build_json().get("status")
```

Comment:
If this is `None`, `if 'config' in status:` below will blow up.

Reply:
well not sure if it is possible for status of build to be None, but ok will make it default {}